docs/overview/getting-started/virtual-machines.md

```diff
@@ -0,0 +1,99 @@
+# Virtual Machines
+
+Avalanche allows developers to implement their Dapp in the language of their choice using the 
+Virtual Machine (VM) framework. VMs define blockchains, and serve as application-level blueprints 
+for how blockchains are created.
+
+Developers can implement VMs in any language, and use libraries and tech stacks that they're
+familiar with. Developers have fine control over the behavior of their blockchain, and can redefine
+the rules of a blockchain to fit any use-case they have.
+
+## Validators
+
+All Avalanche validators as members of the Avalanche primary network are required to run three VMs:
+
+- Coreth: Defines the Contract Chain (C-Chain); supports smart contract functionality and is
+- EVM-compatible.
```

Comment:
Not sure if this should be on its own line, as it is still about Coreth.

Reply:
Yeah my IDE formatted this weird. Removed the bullet.